string trigger
---
string pictureName
bool good